src/duckx.hpp

```diff
@@ -8,6 +8,7 @@
 #define DUCKX_H
 
 #include <stdlib.h>
+#include <cstdio>
```

Comment:
```suggestion

```

@angguss Also, I think we don't need to add this header file. I removed this header file and compiled DuckX and then I saw everything works perfectly.

Reply:
Without this header the build fails in to build in TravisCI - the original PR didn't include it

Reply:
@angguss Can you describe to me why build fails in TravisCI?